data_source_obmcs_database_databases_test.go

```diff
@@ -11,25 +11,67 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
-type DatabaseDatabasesTestSuite struct {
+type DatasourceDatabaseDBSystemTestSuite struct {
 	suite.Suite
 	Client    *baremetal.Client
 	Config    string
 	Provider  terraform.ResourceProvider
 	Providers map[string]terraform.ResourceProvider
 }
 
-func (s *DatabaseDatabasesTestSuite) SetupTest() {
+func (s *DatasourceDatabaseDBSystemTestSuite) SetupTest() {
 	s.Client = testAccClient
 	s.Provider = testAccProvider
 	s.Providers = testAccProviders
-	s.Config = databaseConfig
+	s.Config = testProviderConfig() + `
+	data "oci_identity_availability_domains" "ADs" {
+		compartment_id = "${var.compartment_id}"
+	}
 
-	s.Config += testProviderConfig()
+	resource "oci_core_virtual_network" "t" {
```

Comment:
can we name the resource something meaningful instead t, just like ADs

Reply:
These are fairly consistently named "t" in the acceptance tests, if we want to go with a new convention we should plan for it, especially in the context of code generating the tests. These names aren't particularly meaningful in the test context anyway since you typically don't create more than one of a thing, and references require the complete resource name prefix already.